src/Services/ExportUtils.vala

```diff
@@ -45,17 +45,17 @@ namespace Quilter.Services.ExportUtils {
     }
 
     public static File? export_pdf (string? file_path = null) {
-        var settings = AppSettings.get_default ();
+        
         int type_of_mode = 0;
 
-        if (settings.dark_mode) {
-            settings.dark_mode = false;
+        if (gsettings.get_boolean("dark-mode")) {
+            gsettings.set_boolean("dark-mode", false);
             type_of_mode = 1;
-        } else if (settings.moon_mode) {
-            settings.moon_mode = false;
+        } else if (gsettings.get_boolean("moon-mode")) {
+            gsettings.set_boolean("moon-mode", false);
             type_of_mode = 2;
-        } else if (settings.sepia_mode) {
-            settings.sepia_mode = false;
+        } else if (gsettings.get_boolean("sepia-mode")) {
```

Comment:
If these options are mutually exclusive, this should be an `enum` not several `boolean` values

Reply:
In another commit outside of the scope of this PR I'm going to do this.